### PR TITLE
Java Arrays (gast_to_code and code_to_gast)

### PR DESCRIPTION
**Pull request by jackdavidweber**

### CODE REVIEW

File: java/gast_to_code/java_helpers.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+"""
+Returns the string representing type of the items in the array.
+Returns error if multiple types in same array or if array is empty
+"""
+def arr_type_helper(gast_arr):
+    if len(gast_arr) == 0:
+        return "impossibleTranslationError: direct translation does not exist"
+    print(gast_arr)
```
Comment (cjoshea9) [inline java/gast_to_code/java_helpers.py:29]: Nit: get rid of print statement

File: java/gast_to_code/gast_to_code_java.py [hunk 2]
```diff
@@ -31,23 +32,23 @@ def handle_log_statement(gast):
...
-        else:
-            kind = "customType"
+        kind = java_helpers.gast_to_java_type(gast["varValue"])
  
         return kind + " " + var_id + " = " + var_value
 
     def handle_aug_assign(gast):
         pass
 
+    # FIXME (taiga#149) gast_to_code should not be able to return System.out.println(1, 2)
```
Comment (augerm) [inline java/gast_to_code/gast_to_code_java.py:43]: nit/optional - Internally we usually format like TODO(taiga#149).

File: java/gast_to_code/gast_to_code_java.py [hunk 3]
```diff
@@ -31,23 +32,23 @@ def handle_log_statement(gast):
...
     def handle_func_call(gast):
+        # handles logstatement for single array
+        if gast["value"]["type"] == "logStatement" and len(gast["args"]) == 1 and gast["args"][0]["type"] == "arr":
```
Comment (augerm) [inline java/gast_to_code/gast_to_code_java.py:46]: Any reason we can't support multiple arguments?

Comment (jackdavidweber) [reply java/gast_to_code/gast_to_code_java.py:46]: Multiple arguments not supported for java: https://stackoverflow.com/questions/25451719/java-system-out-println-syntax-help-for-multiple-variables/25451829

**Pull request merged**